### ISSUE TITLE
error when printing qr code 

### ISSUE THREAD

**Issue by appcrea** [setchkin/Litecoin-Poster-Generator#1]
hello I have tried your work, but when printing the pdf, the qr code is not displayed correctly ... it has a solution? 

Thanks

**Comment by setchkin** [comment]
Can you explain what OS, browser, address format and wallet you used?

It may be the issue the QR code only gives you the entered address but the wallet software you are using is configured for LTC:Address type of input from the QR.

Can you Try to enter the adress format as LTC:Address to the address input and try to generate again?

**Comment by appcrea** [comment]
hello thank you for your reply 

I have made a test from its demo link: 

https://setchkin.github.io/Litecoin-Poster-Generator/

I have used as proof of Wallet Address:
LYCofhMWSu7B6wfGvUurXyFCRS3NtynZzF

the pdf shows the qr code cut out 

I have performed the tests from the web browser: chrome, from my pc and laptop .

It is that if I enter your example using my mobile phone, the qr code in the pdf is displayed correctly .

**Comment by appcrea** [comment]
hello, have you been able to see it

**Comment by setchkin** [comment]
> hello, have you been able to see it 

Yes. I understand what you mean now. I checked with Brave, Vivaldi and Firefox too. Brave also had this problem you saw. It was cutting the upper part of the QR code. The thing is it ruined my other poster generators too.

I am only a hobbyist and not a proffessional programmer so I have no idea why this happens right now but I will try to ask and solve this problem to some proffessionals. 

In the meantime I can suggest you to use Firefox, Vivaldi or your mobile browser.
The problem most probably occurs due to some change in Chrome I guess but I still need a professional help.

If you ever manage to find the solution feel free to tell me or fork the project if you wish.

**Comment by appcrea** [comment]
thanks, but i'm not a programmer :(
